README.md

```diff
@@ -1 +1,37 @@
-# node-patterns-concurrency-lab1
+# Async Lab
+
+## Objectives
+
+1. Implement your own async loop
+1. You cannot use async or some other library
+1. You need to make 10 requests to another service and wait until all of them responded before rendering
```

Comment:
I think specifying "10 requests" makes it difficult for students to really see what they're being asked to do. Maybe we could reword the objective so that it's something like "Make a series of requests to another service and wait until all of them have responded before rendering results"